src/index.html

```diff
@@ -1,13 +1,72 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="styles/index.css">
+  </head>
+  <body>
+    <section class="item">
+      <article
+        class="product-card item__product-card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+          class="product-card__image"
+        >
+        <p
+          class="product-card__description"
+        >
```

Comment:
```suggestion
        <p  class="product-card__description">
```
there is no need in that, fix in other places

Reply:
fix this 
![image](https://user-images.githubusercontent.com/99331793/203063254-e3eef631-54eb-46a4-bf3c-692fae99e5b7.png)